formatting/availability_zones.go

```diff
@@ -0,0 +1,22 @@
+package formatting
+
+import (
+	"sort"
+	"strings"
+)
+
+// AvailabilityZonesList returns the list of availability zones
+// as one string consisting of uppercase letters only, separated by comma.
+// Example: "A,B,C".
+func AvailabilityZonesList(az []string) string {
```

Comment:
Do we forsee ever having an `az` list of, for example, `eu-west1a, eu-west1b, eu-west2a`? And if so how do we expect to handle this?

Reply:
Having both `eu-west1` and `eu-west2` in the same list is no considered currently, as these would be different regions, which is not supported by our product.